pkg/apiclient/api/openapi.yaml

```diff
@@ -1774,6 +1774,7 @@ components:
       type: object
     GitStatus:
       example:
+        behind: 6
```

Comment:
also what can we do for these hiccups

Reply:
What do you mean?

Reply:
behind: 6  
          uptime: 1      behind: 6    in this file pkg/apiclient/api/openapi.yaml

Reply:
That's fine 😄 Those are just auto-generated examples